src/WpfMath/TexFormula.cs

```diff
@@ -33,7 +33,6 @@ public TexRenderer GetRenderer(TexStyle style, double scale, string systemTextFo
         public void Add(TexFormula formula, SourceSpan source = null)
         {
             Debug.Assert(formula != null);
-            Debug.Assert(formula.RootAtom != null);

```

Comment:
I don't think you need to remove these assertions now (when the new code produces non-nullable items for these formulas), do you?

Reply:
Ok